oak/server/rust/src/asylo_alloc.rs

```diff
@@ -0,0 +1,34 @@
+use core::alloc::{GlobalAlloc, Layout};
+
+extern "C" {
+    // See https://www.sourceware.org/newlib/libc.html#malloc.
+    fn malloc(nbytes: usize) -> *mut u8;
+    fn memalign(align: usize, nbytes: usize) -> *mut u8;
+    fn free(aptr: *mut u8);
+}
+
+// Aligment is defined in the Intel SGX SDK.
+// See https://github.com/intel/linux-sgx/blob/master/sdk/tlibc/stdlib/malloc.c#L541.
```

Comment:
The link here will rot over time; maybe something like "See the definition of `MALLOC_ALIGNMENT` for `_TLIBC_` in https://github.com/intel/linux-sgx/blob/master/sdk/tlibc/stdlib/malloc.c"

Reply:
Done.